referencemodels/src/test/java/org/openehr/referencemodels/BmmComparison.java

```diff
@@ -85,6 +85,11 @@ public BmmComparison(Set<String> extraParamsInModel, Map<String,
             if(!isIgnorableModelParam(classDefinition.getName(), attributeInfo.getRmName())) {
                 BmmProperty<?> bmmProperty = classDefinition.getFlatProperties().get(attributeInfo.getRmName());
                 if (bmmProperty == null) {
+                    if (attributeInfo.getTypeNameInCollection().toLowerCase().startsWith("bool")) {
+                        bmmProperty = classDefinition.getFlatProperties().get("is_" + attributeInfo.getRmName());
+                    }
+                }
```

Comment:
I think we added the `@RMProperty` annotation in the RM to solve these problems. Would that also be an option for the AOM?